Makefile

```diff
@@ -368,6 +391,11 @@ $(INSTALLER).raw: $(EFI_PART) $(ROOTFS_IMG) $(INITRD_IMG) $(CONFIG_IMG) $(PERSIS
 $(INSTALLER).iso: $(EFI_PART) $(ROOTFS_IMG) $(INITRD_IMG) $(CONFIG_IMG) $(PERSIST_IMG) | $(INSTALLER)
 	./tools/makeiso.sh $| $@
 
+$(LIVE).parallels: $(LIVE).raw
+	rm -rf $@; mkdir $@
+	qemu-img convert -O parallels $< $@/live.0.$(PARALLELS_UUID).hds
```

Comment:
Before you convert you need to make sure you resize the live disk to at `$(MEDIA_SIZE)` otherwise the resulting VM is useless since it has no space for /persist partition.

Reply:
Ok, I will resize the disk to `$(MEDIA_SIZE) `